vars/runATH.groovy

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env groovy
```

Comment:
This shebang is conceptually wrong IMO, and from a user standpoint we should probably not encourage people to think/remember this is Groovy. (this is not really, because of the subtle differences with groovy-cps)

Reply:
`buildPlugin` includes a shebang and is also cited in the  [doc|https://jenkins.io/doc/book/pipeline/getting-started/#defining-a-pipeline-in-scm] as a way to make sure you IDE understands the syntax.

So, for the moment I would prefer not to remove it, I understand your point but for me IDE support is a must in pipeline development given all the pending improvements.